docs/concepts.md

```diff
@@ -25,7 +25,7 @@ libraries. See [Writing Tests](/docs/writing-tests/) for more information.
 
 A distributable set of tests and tools built and packaged
 using Inferno to help testers evaluate the conformance of a system to
-FHIR base specification requirements, relevant FHIR Implementation
+FHIR-based specification requirements, relevant FHIR Implementation
```

Comment:
I think this was actually right previously, I think it's saying it checks the conformance of a system to the base FHIR specification. Maybe change this to "base FHIR specification" for clarity?

Reply:
That makes sense, I wasn't sure which way to interpret it!  I implemented your suggestion.